ansible/roles/jenkins-seed-job/templates/seed_job_config.xml.j2

```diff
@@ -13,7 +13,7 @@
         <hudson.model.StringParameterDefinition>
           <name>REPOSITORY_URL</name>
           <description>Repository to download the DSL job descriptors.</description>
-          <defaultValue>https://github.com/{{pipeline_constants.GITHUB_ORGANIZATION_NAME}}/infrastructure.git</defaultValue>
+          <defaultValue>https://{{pipeline_constants.GITHUB_DOMAIN}}/{{pipeline_constants.GITHUB_ORGANIZATION_NAME}}/infrastructure.git</defaultValue>
         </hudson.model.StringParameterDefinition>
```

Comment:
GITHUB_DOMAIN needs to be added as a parameter in this file. You can place it right before GITHUB_ORGANIZATION_NAME

Reply:
The SSH key is not being used because the host `github.com` no longer matches the entry `github` from the config file. You'll have to replace every occurrence of `ssh://git@github.com/` to `ssh://git@github/`.
I'd also change every HTTPS URL to SSH, to make use of the same configuration and detect authorization issues earlier. With this, you can also remove my suggestion above and the variable `{{pipeline_constants.GITHUB_DOMAIN}}` from this file.